pyaccel/optics.py

```diff
@@ -463,8 +463,17 @@ def get_frac_tunes(accelerator=None, m66=None, closed_orbit=None,
 
 
 @_interactive
-def get_chromaticities(accelerator):
-    raise OpticsException('not implemented')
+def get_chromaticities(accelerator, energy_offset=1e-6):
+    acc = accelerator[:]
```

Comment:
I think it is better to avoid making copies.
Store the initial values of `radiation_on` and `cavity_on` to recover the accelerator state later.